index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="styles.css">
+  <title>Hello Microverse</title>
+</head>
+<body>
+ <h1 class="heading">Hello Microverse</h1>
```

Comment:
- [OPTIONAL] - I recommend using the <header> tag for class if there are sub heading it helps you to group them to style it and makes your project more readable and accessible. You will learn that in the future lessons, but you can check more about that [here](https://www.searchenginejournal.com/on-page-seo/header-tags/#close)